mahta/package maneger/package_maneger.txt

```diff
@@ -0,0 +1,21 @@
+In learning to work with package manegers, I first had to choose which one I'm going to use. based on a few articles read I chose yarn for better performance and being faster than npm, and more popular than pnpm.
```

Comment:
<img width="991" alt="Screenshot 2024-03-18 at 9 16 40 AM" src="https://github.com/ConnecMent/momentum/assets/63106898/0bd0f5a9-c63b-4056-a3c7-16e428565501">

Reply:
`Bun` has been recently published 
and is not recommended for use in production mode, which is why they're utilizing `pnpm`. Please also execute the following commands: `pnpm update` and 'Install React version 16.8.2'."

Reply:
where is pnpm-lock.yml?
do you work with pnpm?